### PR TITLE
Fixes multiple Wishlist issues.

### PR DESCRIPTION
**Pull request by inputsh**
- elementary 0.4 might not be named Loki.
- I have no idea what the hell does `Official elementary application` mean and why it is there (two times).
- Improved the info in a couple of places.
- Closed the parenthesis.

### CODE REVIEW

**Comment by quassy** [comment]
"Official elementary application." just means it was developed by the elementary (apps) team and shipped as part of Jupiter (I believe). So this was an indication of official but abandoned elementary apps in the hopes someone would continue the work. Will take a look at the rest later, thanks for the PR!

**Comment by inputsh** [comment]
Oh, thanks. I did not know that.

Maybe then changing it to "Was an official elementary application"?

File: pages/wishlist.md [hunk 1]
```diff
@@ -6,28 +6,28 @@ permalink: /wishlist/
...
 
-If you are a developer please squash some [elementary bugs](http://bugs.launchpad.net/elementary) or help out [other developers]({{ site.baseurl }}/apps/) first. Or do you own thing: Any item on this list is a good place to start your own elementary project. 
+If you are a developer please squash some [elementary bugs](http://bugs.launchpad.net/elementary) or help out [other developers]({{ site.baseurl }}/apps/) first. Or do you own thing: Any item on this list is a good place to start your own elementary project.
 
 If there is another app for elementary you are missing desperately, feel free to [file an issue at GitHub](https://github.com/quassy/elementary-apps/issues/new) to get it added to the list (or develop it yourself and tell us about it). 
 
 Name | Generic | Info
 ---|---|---
-[AppCenter](https://launchpad.net/appcenter) (in-development) | Software Store | Official elementary app. Only included because people often ask: This in development, but it is not expected to be finished before elementary 0.4 Loki.
```
Comment (quassy) [inline pages/wishlist.md:15]: While not officially announced Loki was floating around for some time and is being used on Launchpad by elementary itself. Even if it does not become the release name, it certainly is the current development codename, similar to Longhorn or Treshold for Vista and 10.

**Comment by quassy** [comment]
I hope I didn't miss anything and I adressed everything you raised with your PR?!

File: pages/wishlist.md [hunk 2]
```diff
@@ -6,28 +6,28 @@ permalink: /wishlist/
...
 
 Name | Generic | Info
 ---|---|---
-[AppCenter](https://launchpad.net/appcenter) (in-development) | Software Store | Official elementary app. Only included because people often ask: This in development, but it is not expected to be finished before elementary 0.4 Loki. 
+[AppCenter](https://launchpad.net/appcenter) (in-development) | Software Store | Official elementary app. Only included because people often ask: This in development, but it is not expected to be finished before elementary 0.4 version. 
 [Synposis]({{ site.baseurl }}/Synposis/) (abandoned) | Archive manager | elementary needs something to replace `file roller` that looks out of place
 *Auth App* | Two-factor code generator | A generator for two-factor authentication codes similar to WinAuth
-[Dexter](https://launchpad.net/dexter-contacts) (abandoned) | Contact Management | Official elementary application. Should allow syncing with contacts in cloud (Google, Microsoft, Apple, self-hosted).
```
Comment (quassy) [inline pages/wishlist.md:18]: I have changed the wording in my commit to make clearer what I meant.

> Abandoned elementary application, was part of elementary 0.1 Jupiter. ...

File: pages/wishlist.md [hunk 3]
```diff
@@ -6,28 +6,28 @@ permalink: /wishlist/
...
-*Indicator Standby Timer* | Indicator | An indicator that allows the user to set X minutes after which the system automatically goes into standby
-[Lingo](https://launchpad.net/lingo-dictionary) (abandoned) | Dictionary App | Official elementary application. Integration with a web-based dictionary like [dict.cc](http://dict.cc) could help with having big and updated database.
-*Messenger* | Multi-messenger | Multi-messenger, basically Pidgin (pretty good) only with native integration (GTK3, headerbars, ...)
-*Office Suite* | Office Suite | By popular demand, but probably an undertaking as big as the whole elementary project. [Writer]({{ site.baseurl }}/Writer/) is the only currently existing app that could be part of this, a spreadsheet and presentation app only exist as mockups so far. 
+*Indicator Night* | Indicator | An indicator adjusting the screen color to time of day, like `f.lux`. Maybe also include the option to switch to dark theme depending on time of day.
```
Comment (quassy) [inline pages/wishlist.md:21]: As _f.lux_ is a name not a piece of code I have opted for italics. _Yes, I really like inline code comments ;)_

**Pull request closed without merging**